WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -43,8 +44,17 @@ - (BOOL)fb_isVisible
   CGSize screenSize = FBAdjustDimensionsForApplication(app.frame.size, (UIInterfaceOrientation)[XCUIDevice sharedDevice].orientation);
   CGRect screenFrame = CGRectMake(0, 0, screenSize.width, screenSize.height);
   BOOL rectIntersects = CGRectIntersectsRect(self.visibleFrame, screenFrame);
-  BOOL isActionable = CGRectContainsPoint(app.frame, self.hitPoint);
-  return rectIntersects && isActionable;
+  @try {
+    BOOL isActionable = CGRectContainsPoint(app.frame, self.hitPoint);
+    return rectIntersects && isActionable;
+  } @catch (NSException *e) {
```

Comment:
I am sorry for the problem, but 
1) Can you split this diff to `shown` -> `visible` change and another one with `try`.
2) On 'try' diff add check whether you want to handle error on that manner. Same as we have `shouldUseTestManagerForVisibilityDetection` (default it to whatever you think is best). It would be easier for me to work on the fix for it.